nixos/tests/systemd-confinement.nix

```diff
@@ -34,52 +34,53 @@ import ./make-test.nix {
         };
       } // removeAttrs config [ "confinement" "serviceConfig" ];
 
-      __testSteps = lib.mkOrder num ''
-        subtest '${lib.escape ["\\" "'"] description}', sub {
-          $machine->succeed('echo ${toString num} > /teststep');
-          ${testScript}
-        };
-      '';
+      __testSteps = lib.mkOrder num (''
```

Comment:
Can we inline whatever this is doing? I assume the outcome will be much more understandable, instead of using nix as a template language here to write python code…

Reply:
done

Reply:
I meant, let's just get rid of `__testSteps` alltogether, it doesn't do much.

Reply:
Just tried to, but it doesn't get any prettier.

Reply:
Alright - let's keep it that way for now, better than having it in perl ;-)